test/unittest/StatisticsBackend/IsActiveTests.cpp

```diff
@@ -60,6 +60,7 @@ class is_active_tests : public ::testing::Test
 
         // Simulate that the backend is monitorizing the domain
         std::shared_ptr<details::Monitor> monitor = std::make_shared<details::Monitor>();
+        monitor->id = domain->id;
```

Comment:
Please see previous comment regarding specific constructor receiving the id.

Reply:
For Monitor refactor along with other needed refactors